.gitignore

```diff
@@ -19,3 +19,4 @@ Session.vim
 
 .pre-commit-config.yaml
 config.toml
+test_config.toml
```

Comment:
isn't this used in tests?

Reply:
it is, but I need compiler_list_url to be `binaries.soliditylang.org/macosx-amd64/list.json` always (in `config.toml` too) and it would be just convinient for me not to commit this file

Reply:
there is a concept of local and public gitignore https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally